test_data/provider_defs/chronicledb.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "EdrDataProviderDefinition",
+  "id": "2579a45e-a2c3-4fdb-946a-d98d9d0eacc4",
+  "name": "ChronicleDB Demo",
+  "baseUrl": "https://dbs-demo.mathematik.uni-marburg.de/dev/cep/edr",
```

Comment:
If I'm not in the uni network, I get no error, but also no data: When should the data be available?

Reply:
I don't understand that. If one isn't in the uni network, it should throw an error. What request do you make?

Reply:
```

GET | http://localhost:4200/api/layers/collections/2579a45e-a2c3-4fdb-946a-d98d9d0eacc4/collections?offset=0&limit=5
-- | --

```

leads to

```
{"error":"EdrInvalidMetadataFormat","message":"The response from the EDR server does not match the expected format."}
```

Reply:
Throwing this error when you are not in the uni network is designed behaviour, because the server returns some html error page instead of the expected json.